src/isolation/filemap.rs

```diff
@@ -17,7 +17,7 @@ use crate::isolation::{
 /// Wrapper around a `HashMap` to map guest paths to arbitrary host paths and track file descriptors.
 #[derive(Debug)]
 pub struct UhyveFileMap {
-	files: HashMap<String, OsString>,
+	files: HashMap<String, PathBuf>,
```

Comment:
actually, the "natural" type for the key here would be `CString`, as it can't/shouldn't ever contain null-bytes.

Reply:
turns out trying to use `CString` here would be overly complicated.

Reply:
yep, that is why I didn't go for CString to begin with and went for the "second closest option", that being OsString.